### PR TITLE
WLSSB: Update Mine, Fix Stuff, Nerf Peri

### PR DESCRIPTION
**Pull request by Lycanium-Z**

### CODE REVIEW

File: mods/wlssb/abilities.js [hunk 1]
```diff
@@ -327,4 +327,21 @@ exports.BattleAbilities = {
...
+		desc: "If Hail is active, this Pokemon's evasiveness is multiplied by 13. This Pokemon takes no damage from Hail.",
+		shortDesc: "If Hail is active, this Pokemon's evasiveness is 3x; immunity to Hail.",
+		onImmunity: function (type, pokemon) {
+			if (type === 'hail') return false;
+		},
+		onModifyAccuracy: function (accuracy) {
+			if (typeof accuracy !== 'number') return;
+			if (this.isWeather('hail')) {
+				this.debug('Snow Cloak - decreasing accuracy');
+				return accuracy * 0.33;
```
Comment (Lycanium-Z) [inline mods/wlssb/abilities.js]: Alright How much should i raise this?

**Comment by Lycanium-Z** [comment]
Tested and works @hoeencoder

File: mods/wlssb/moves.js [hunk 2]
```diff
@@ -440,63 +440,24 @@ exports.BattleMovedex = {
...
+			this.setWeather('hail');
+			this.useMove('leechseed', pokemon);
+			this.useMove('foresight', pokemon);
 		},
-		target: "normal",
-		type: "Rock",
-		zMovePower: 180,
-		contestType: "Tough",
+		secondary: false,
+		target: "self",
```
Comment (DeathlyPlays) [inline mods/wlssb/moves.js]: Why make it Self when you are using Foresight & Leech Seed on foe

File: mods/wlssb/moves.js [hunk 3]
```diff
@@ -711,7 +672,7 @@ exports.BattleMovedex = {
 			this.add('-anim', source, "Future Sight", target);
 			this.add('-anim', source, "Psycho Boost", source);
 		},
-		desc: "Raises all stats by 1 and opponent falls asleep.",
+		desc: "50% chance to all stats by 1 and opponent falls asleep.",
```
Comment (DeathlyPlays) [inline mods/wlssb/moves.js]: Add raise after to

File: mods/wlssb/moves.js (hunk 2 above)
Comment (Lycanium-Z) [reply mods/wlssb/moves.js]: Makes Sure the move doesn’t miss

Comment (HoeenCoder) [reply mods/wlssb/moves.js]: thats, not the appropriate way to ensure a move hits, use ``accuracy: true,`` for that

Comment (HoeenCoder) [reply mods/wlssb/moves.js]: wont this make you use the moves on yourself? not sure

Comment (Lycanium-Z) [reply mods/wlssb/moves.js]: No

Comment (Lycanium-Z) [reply mods/wlssb/moves.js]: I tested it and it does use it on the opponent

File: data/wlssb-data.txt [hunk 4]
```diff
@@ -12,15 +12,18 @@ Notes: N/A.
...
+@ Focus Sash
+EVs: None
+Impish Nature
+Level 1
+Ability: Extreme Snow Cloak
+    - Evasion is multiplied by 2 during hail.
+Innate Ability: Snow Warning
+Moves: Endeavor, Stealth Rock, Ice Shard
+Signature Move: Unserious
+	- Ice Type. Forces weather to hail, uses foresight.
```
Comment (DeathlyPlays) [inline data/wlssb-data.txt]: Leech Seed also

Comment (DeathlyPlays) [reply data/wlssb-data.txt]: Nvm u removed that

Comment (Lycanium-Z) [reply data/wlssb-data.txt]: Yes

**Pull request merged**